packages/sitecore-jss-nextjs/src/editing/render-middleware.test.ts

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable dot-notation */
 import chai from 'chai';
 import chaiString from 'chai-string';
+import { QUERY_PARAM_EDITING_SECRET } from '@sitecore-jss/sitecore-jss/editing';
 import { RenderMiddlewareBase } from './render-middleware';
 import {
-  QUERY_PARAM_EDITING_SECRET,
   QUERY_PARAM_VERCEL_PROTECTION_BYPASS,
```

Comment:
Do we plan supporting vercel protection for Angular? Should these be moved to base package too if we do?

Reply:
@art-alexeyenko Correct, I had the same thought as you and I didn't hear anything regarding that, let's leave it as it's for now. If we will plan something in this direction, we will need to reuse that